catalyst_voices/apps/voices/integration_test/app_test.dart

```diff
@@ -30,96 +27,101 @@ void main() async {
     await restartDependencies();
   });
 
-  patrolWidgetTest(
-    'Spaces drawer - visitor - no drawer button',
-    (PatrolTester $) async {
-      await $.pumpWidgetAndSettle(App(routerConfig: router));
-      await $(OverallSpacesPage.visitorShortcutBtn)
-          .tap(settleTimeout: const Duration(seconds: 10));
-      expect($(AppBarPage.spacesDrawerButton).exists, false);
-    },
-  );
+  group(
+    'Spaces drawer -',
+    () {
+      patrolWidgetTest(
+        'visitor - no drawer button',
+        (PatrolTester $) async {
+          await $.pumpWidgetAndSettle(App(routerConfig: router));
+          await $(OverallSpacesPage.visitorShortcutBtn)
+              .tap(settleTimeout: const Duration(seconds: 10));
+          expect($(AppBarPage.spacesDrawerButton).exists, false);
+        },
+      );
 
-  patrolWidgetTest(
-    'Spaces drawer - guest - chooser - clicking on icons works correctly',
-    (PatrolTester $) async {
-      await $.pumpWidgetAndSettle(App(routerConfig: router));
-      await $(OverallSpacesPage.guestShortcutBtn)
-          .tap(settleTimeout: const Duration(seconds: 10));
-      await $(AppBarPage.spacesDrawerButton).waitUntilVisible().tap();
-      SpacesDrawerPage.commonElementsLookAsExpected($);
+      patrolWidgetTest(
```

Comment:
You're sure about using patrol? It doesn't work on web

Reply:
I agree with @dtscalac. Any pubs or libraries must work on Web, iOS and Android.

Reply:
Yes, we're using only patrol finders with pure flutter, so still running tests the same flutter way 'flutter drive ...' not 'patrol test ...'

Reply:
Makes sense, thanks for clarification.